polyfill/lib/intl.mjs

```diff
@@ -118,7 +117,7 @@ function amend(options = {}, amended = {}) {
 function timeAmend(options) {
   options = amend(options, { year: false, month: false, day: false });
   if (!hasTimeOptions(options)) {
-    options = ObjectAssign(options, {
+    options = Object.assign(options, {
```

Comment:
Note that if you want these to be robust against runtime env modification, you still have to cache the function at module level.